library/src/main/java/com/telefonica/mistica/compose/card/postercard/PosterCard.kt

```diff
@@ -0,0 +1,94 @@
+package com.telefonica.mistica.compose.card.postercard
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.telefonica.mistica.compose.tag.Tag
+import com.telefonica.mistica.compose.theme.MisticaTheme
+
+@Composable
+fun PosterCard(
+    modifier: Modifier = Modifier,
+    aspectRatio: PosterCardAspectRatio,
+    backgroundType: PosterCardBackgroundType,
+    headline: Tag? = null,
+    preTitle: String? = null,
+    title: String? = null,
+    subtitle: String? = null,
+    description: String? = null,
+    firstTopAction: TopActionData? = null,
+    secondTopAction: TopActionData? = null,
+    onClickAction: (() -> Unit)? = null,
+    customContent: (@Composable () -> Unit)? = null,
+) {
+    BoxWithConstraints(modifier = modifier) {
+        androidx.compose.material.Card(
+            elevation = 0.dp,
+            backgroundColor = Color.Red,
+            shape = RoundedCornerShape(16.dp),
```

Comment:
According to Figma spec, should be "container" value instead of fixed 16.dp right?
https://www.figma.com/design/tKdPOfcUALzVIh5oizFbm7/%F0%9F%94%B8-Cards-Specs?node-id=2085-7952&t=7A6BCRoDU8omYQoY-4
```suggestion
            shape = RoundedCornerShape(MisticaTheme.radius.containerBorderRadius),
```